src/online-members/sse-events.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+// setupDragDropEvent('.members-list');
+
+let currentOnlineList = [];
+let previousOnlineList = [];
+
+function setUpOnlineMembersEventSource(url) {
+  const eventSource = new EventSource(url, { withCredentials: true });
+
+  eventSource.onopen = (e) => {
+    console.info('The connection has been established.');
+  };
+
+  eventSource.onmessage = function (event) {
+    try {
+      const objectData = JSON.parse(event.data);
+      currentOnlineList = objectData.users;
```

Comment:
@prakashchoudhary07 , is this causing this issue where we are looping through null data?